sdk/src/main/java/org/coralibre/android/sdk/internal/device_info/ConfidenceLevel.java

```diff
@@ -0,0 +1,49 @@
+package org.coralibre.android.sdk.internal.device_info;
+
+import android.content.ComponentName;
+
+import java.util.Enumeration;
+
+public enum ConfidenceLevel {
+    NONE,
+    LOW,
+    MEDIUM,
+    HIGH;
+
+    private static final  int LOW_INT = 1;
+    private static final int MEDIUM_INT = 2;
+    private static final int HIGH_INT = 3;
+    private static final String LOW_STR = "LOW";
+    private static final String MEDIUM_STR = "MEDIUM";
+    private static final String HIGH_STR = "HIGH";
+    private static final String NONE_STR = "NONE";
+
+    public static ConfidenceLevel getConfidenceLevel(int raw) throws EnumConstantNotPresentException {
```

Comment:
Can `EnumConstantNotPresentException` be thrown in this method? I don't think so